airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -52,6 +55,14 @@
           }
         ]
       },
+      "start_date": {
+        "type": "string",
+        "title": "Start Date",
+        "description": "The date from which you'd like to replicate data for GitHub in the format YYYY-MM-DDT00:00:00Z. All data generated after this date will be replicated.",
```

Comment:
Thanks for clarifying the streams not supported piece.

Can we update this description to call this out then and link to docs?

The date from which you'd like to replicate data for GitHub in the format YYYY-MM-DDT00:00:00Z. This field doesn't apply to all streams, see [the docs] for more info